elastic/beats
# Fixing broken documentation links for filebeat and metricbeat

## What does this PR do?
This PR fixes `filebeat` and `metricbeat` broken links that were directed to the wrong branch (`main` instead of `master`).
This resulted in invalid `404` pages when attempting to access the document resources.

## Why is it important?
In order to see the documentation and not browse to broken links.

## Checklist

## Author's Checklist

## How to test this PR locally
Browse the documentation links and verify that they are directed to the relevant pages.

## Related issues
- 

## Use cases
As a developer/user, I would like to get up-to-date valid documentation links.

## Screenshots

## Logs
